ecl/hql/hqltrans.cpp

```diff
@@ -375,19 +414,20 @@ static void insertScopeSymbols(IHqlScope * newScope, HqlExprArray const & symbol
 
 //---------------------------------------------------------------------------
 
-static HqlTransformerBase * activeTransformer;      // not thread safe
+static __thread HqlTransformerBase * activeTransformer = nullptr;
```

Comment:
That's why we have a define in platform.h:

#define __thread __declspec(thread)